bulk_create.go

```diff
@@ -0,0 +1,44 @@
+package opensearchtools
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// Create encapsulates a bulk Create Action.
+// Creates a document if it doesn’t already exist and returns an error otherwise.
+//
+// For more details, see https://opensearch.org/docs/latest/api-reference/document-apis/bulk/
+type Create struct {
+	doc RoutableDoc
```

Comment:
Is is really a bulk operation if it can only create a single document?  Probably should just be `create.go`.